CHANGELOG.mdx

```diff
@@ -2324,3 +2324,7 @@ Prefix the change with one of these keywords:
 ### Changed
 
 - Made social login buttons optional on login page
+
+### Added
+
+- New component named Carleton360
```

Comment:
This needs to be place with the unreleased notes, you accidentally placed at bottom of the changelog